### PR TITLE
Functional tests novaclient

### PR DESCRIPTION
**Pull request by vmoravec**

### CODE REVIEW

**Comment by toabctl** [comment]
Failed due to some unrelated things. But our code was not tested. Need to fix the gate first.

**Comment by rsalevsky** [comment]
CCT run succeeds, but as far as I can see the tests didn't run.

File: features/step_definitions/functional_tests/functional_tests_steps.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    public: true
+  )
+
+  wait_for "Image status set 'active'", max: "60 seconds", sleep: "2 seconds" do
+    image = control_node.openstack.image.show(@test_image.id)
+    break if image.status == "active"
+  end
+end
+
+Then(/^all the funtional tests for the package "([^"]*)" pass$/) do |package_name|
```
Comment (rsalevsky) [inline features/step_definitions/functional_tests/functional_tests_steps.rb]: typo funtional => functional

**Comment by vmoravec** [comment]
Where is the fix needed for the gate, I can't find the code for it.

**Comment by rsalevsky** [comment]
We run cct with `bundle exec rake features:base` so when I understand your PR right then normally it should run by default.

**Comment by toabctl** [comment]
Superseeded by https://github.com/SUSE-Cloud/cct/pull/24

**Pull request closed without merging**